### DISCUSSION TITLE
Can I get the expected error of the quantity from MOOSE?

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#21841] [Q&A Modules: Phase field] [answered]
Hi everyone,

I am using first order lagrange element in my simulation. I was wondering if MOOSE has any postprocessor that can calculate the expected error of the quantity, such as energy, for me?

Thank you.
Best,

**Comment by GiudGiud** [comment, marked as answer]
Hello

If you know the reference solution, you may compute the error using this postprocessor for example
https://mooseframework.inl.gov/source/postprocessors/ElementL2Error.html
Otherwise you should lead a refinement study (see uniform refinement mesh parameters) to evaluate the error

Guillaume
